lib/android-helpers.js

```diff
@@ -632,7 +632,7 @@ helpers.initDevice = async function (adb, opts) {
   if (opts.unicodeKeyboard) {
     defaultIME = await helpers.initUnicodeKeyboard(adb);
   }
-  if (_.isUndefined(opts.unlockType)) {
+  if (_.isUndefined(opts.unlockType) && !opts.skipUnlock) {
```

Comment:
Do we log somewhere if the user gives an unlock type _and_ sets `skipUnlock` to true?

Reply:
> (who is gonna do such unlogical stuff though?)

you're still asking that question after working on appium for so long?

Reply:
Yes I still do believe in people...